parl/framework/model_base.py

```diff
@@ -0,0 +1,104 @@
+#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Base class to define an Algorithm.
+"""
+
+from abc import ABCMeta, abstractmethod
+from parl.utils.utils import has_fun
+
+__all__ = ['Network', 'Model']
+
+
+class Network(object):
+    """
+    A Network is an unordered set of LayerFuncs or Networks.
+    """
+
+    def sync_paras_to(self, target_net):
+        assert not target_net is self, "cannot copy between identical networks"
+        assert isinstance(target_net, Network)
+        assert self.__class__.__name__ == target_net.__class__.__name__, \
```

Comment:
add infomation "sync_paras_to only support Networks that through deepcopy way"

Reply:
done.The message has been add in other line because actual position to add message is not here.